src/test/java/com/wikia/webdriver/elements/mercury/components/TopBar.java

```diff
@@ -2,15 +2,16 @@
 
 import com.wikia.webdriver.common.core.elemnt.Wait;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.wikia.webdriver.common.core.elemnt.Wait'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)